PR for #57: Benchmarking + Paper

santiagohermo opened the pull request:
Can you quickly peer review #57 @gabrieleborg? See https://github.com/diegogentilepassaro/min_wage_rent/issues/57#issuecomment-716199876 for an explanation of what is taking place in this branch.

The idea is to peer review it quickly so that the paper gets merged to `master`, and we can work on that

gabrieleborg commented:
Will do!

diegogentilepassaro commented:
@gabrieleborg if you want I can take over this PR and you just keep focusing on LODES.

gabrieleborg commented:
@diegogentilepassaro that'd be nice, thanks!

diegogentilepassaro commented:
Perfect @gabrieleborg! I will do it then :D

santiagohermo commented:
Hi @diegogentilepassaro. I wanted to flag that I'm modifying the draft directly in this branch for #67 (so you might find some inconsistencies, which represent work in progress!)

diegogentilepassaro reviewed (commented):
@santiagohermo some very basic initial reactions here. I will keep going by spending some time in reviewing/improving the model if you agree. 

Let me know otherwise how do you think I should proceed!

santiagohermo commented on issue57/header.tex:
```diff
@@ -0,0 +1,14 @@
+\setbeamertemplate{navigation symbols}{}
```
This file is issue by `slides.Rmarkdown` to format the Beamer.

santiagohermo commented on issue57/slides.RMarkdown:
```diff
@@ -0,0 +1,195 @@
+---
+title: "Do Minimum Wages Increase Rents?"
```
Sounds good. Also the idea of having short and long versions is nice. I find making slides in Rmarkdown quite intuitive, so I wouldn't mind to keep them in this format. However, we can also make them in Latex building off of [Gabri's overleaf](https://www.overleaf.com/project/5f5fc495aa64f70001dfc421). What you you guys prefer @diegogentilepassaro @gabrieleborg ?

Once we decide I'll add an proper initial version of the slides to this branch.

diegogentilepassaro replied:
Both sounds good to me! Maybe let's do the short version in markdown and the long one in Latex, given that it is whet we have done so far.

santiagohermo commented on paper/03data_sample.tex:
```diff
@@ -0,0 +1,177 @@
+    \caption{Descriptive statistics and comparison with representative zipcodes}
+    \centering
+    \label{tab:desc_stats}    
+    \input{input/desc_stats.tex}
```
Yes! I added this because the version in `input` has a nicer format (which I fixed manually for the paper Gabri presented to his committee). We have to improve our descriptive table and change this pointer. Should I do this in this branch?

diegogentilepassaro replied:
I see! Thanks for clarification @santiagohermo. To get that fixed before we merge would be great.

santiagohermo commented:
Thanks @diegogentilepassaro! I had a few questions about your inline comments above. Still some things to do in this branch it seems.

I saw that you started the overleaf for the model. Looks great! A few questions about that:
- Following Jesse's comments, should we keep the current model as section 2 of the paper? Maybe we can send it to an appendix, use it in the benchmarking, and once we have a nicer version add it back?
- If we are taking seriously the idea of improving the model we probably want to open a follow-up issue to discuss it there.
- Probably more valid for the future. But after you are done it would be nice to add the notes to `analysis/notes`, so that we keep everything in the repo.

diegogentilepassaro commented:
Thanks to you @santiagohermo! AS for your comments:

- I think that we should remove the model from the paper for now and, as per Jesse's suggestion, we should replace it with a simple calculation. Probably, it will fit after results now instead of section 2. 
- I agree. Let's discuss that tomorrow and open a new issue after we merge this one. 
- Great idea, let's add it now and we can use it as a base for discussion.

santiagohermo commented:
Hey @diegogentilepassaro @gabrieleborg. I did a few more things here, following our discussions. I think that this is a good moment for a review, merge back to master, and continue working on the draft in #65 

Changes:

- I added a basic template of short and long slides in `paper_slides\slides` [here](https://github.com/diegogentilepassaro/min_wage_rent/tree/935e3220342a8884c9f862265a4659eedb32babf/paper_slides/slides). See, for example, [long](https://github.com/diegogentilepassaro/min_wage_rent/blob/935e3220342a8884c9f862265a4659eedb32babf/paper_slides/slides/long_slides.pdf) pdf. I chose a particular style. We can write these slides better and reconsider the style once we have a presentation in the horizon.

- I moved the paper to `paper_slides\paper` [here](https://github.com/diegogentilepassaro/min_wage_rent/tree/935e3220342a8884c9f862265a4659eedb32babf/paper_slides/paper). I sent the model to the appendix in a quick manner (writing has to be improved).

- [`paper_slides/biblio.bib`](https://github.com/diegogentilepassaro/min_wage_rent/blob/935e3220342a8884c9f862265a4659eedb32babf/paper_slides/biblio.bib) contains references and is used both by slides and paper. I think we should organize the references better at some point, and make sure all the papers we want are there (e.g., I added some that gabri had used in this breakfast presentation but wasn't in the paper).

- I fully restructured `analysis/descriptive`:
   - the [code](https://github.com/diegogentilepassaro/min_wage_rent/blob/935e3220342a8884c9f862265a4659eedb32babf/analysis/descriptive/code/build_summary_tables.R) that produces the descriptive table is now written in R. Two reasons to migrate: 
     - it's much easier to do manipulations and format things nicely in R than in STATA
     - we can use [stargazer](https://www.jakeruss.com/cheatsheets/stargazer/)
   - I changed things conceptually. Now we produce two tables: 
     - [`stats_sample.tex`](https://github.com/diegogentilepassaro/min_wage_rent/blob/935e3220342a8884c9f862265a4659eedb32babf/analysis/descriptive/output/stats_sample.tex), which is meant to compare our sample of zipcodes to the US and top CBSA. See how this looks like in [the paper](https://github.com/diegogentilepassaro/min_wage_rent/blob/935e3220342a8884c9f862265a4659eedb32babf/paper_slides/paper/00main.pdf) page 8; and 
     - [`stats_est_panel.tex`](https://github.com/diegogentilepassaro/min_wage_rent/blob/935e3220342a8884c9f862265a4659eedb32babf/analysis/descriptive/output/stats_est_panel.tex), which is supposed to show summary statistics of the variables in the panel we use in regressions. 
     There is a lot of room for improvement here, for example by adding the exact economic controls we use (I added a few for illustration).

- I implemented some other fixes, such as using the monthly cps and deleting unused files. On this, we may want to remove unused code in [`analysis/descriptive`](https://github.com/diegogentilepassaro/min_wage_rent/tree/935e3220342a8884c9f862265a4659eedb32babf/analysis/descriptive/code)


Let me know what you guys think! Once this is done, I'll merge and continue working on improving the paper in #65

gabrieleborg commented:
Thanks @santiagohermo! The changes all make sense to me, and I agree that this is a good point for review and the migrate to #65. As you said, once we move there let's maybe add come appendix table with summary stats for the various econ controls. 

On the `stats_sample.tex` table, that looks fine to me :) the only thing, I would remove the decimals for the median income as they aren't informative and make it harder to read.  

Thanks for all the restructuring!

santiagohermo commented:
Thanks @gabrieleborg! Fully removing digits appears to be a relatively complicated task. Instead, I changed the number of decimal places to 2 throughout, which hopefully makes things a little better. We can work on further improvements later.
